modules/system-health-dashboard-page-overview.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * operating/use-system-health-dashboard.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="system-health-dashboard-page-overview_{context}"]
+= System health dashboard page overview
+
+The *System Health* dashboard page organizes information into the following groups:
+
+* *Cluster status*: The overall status of the cluster, and the status of essential {rh-rhacs-first} components such as Sensor, Collector and Admission Controller.
+
+* *Sensor upgrade*: The status of any pending or in-progress upgrades for Sensor.
+
+* *Credential expiration*: The expiration dates of critical credentials. 
+
+* *StackRox Scanner Vulnerability Definitions*: The status of the vulnerability definitions database that StackRox Scanner uses. 
+
+* *Scanner V4 Vulnerabilities*: The vulnerability information associated with Scanner V4.
+
+* *Image Integrations*: The health and connectivity status of integrated image registries. 
+
+* *Notifier Integrations*: The health of any notifiers, such as Slack, email, Jira, or other similar integrations that you have integrated.
```

Comment:
```suggestion
* *Notifier Integrations*: The health of any notifiers, such as Slack, email, Jira, or other similar integrations.
```

Reply:
Is there a reason this one gets examples but others don't? If you want it to be parallel to backup, could do:
```suggestion
* *Notifier Integrations*: The health of any notifiers that you have integrated.
```